src/iGenVar.cpp

```diff
@@ -59,20 +59,21 @@ struct cmd_arguments
 {
     std::filesystem::path alignment_file_path{};
     std::filesystem::path insertion_file_path{};
+    std::filesystem::path output_file_path{};
     std::vector<detection_methods> methods{cigar_string, split_read, read_pairs, read_depth};   // default: all methods
-    clustering_methods clustering_method{simple_clustering};    // default: simple clustering method
-    refinement_methods refinement_method{no_refinement};        // default: no refinement
+    clustering_methods clustering_method{simple_clustering};                                    // default: simple clustering method
+    refinement_methods refinement_method{no_refinement};                                        // default: no refinement
     uint64_t min_var_length = 30;
 };
 
 void initialize_argument_parser(seqan3::argument_parser & parser, cmd_arguments & args)
 {
-    parser.info.author = "David Heller & Lydia Buntrock";
+    parser.info.author = "Lydia Buntrock, David Heller, Joshua Kim";
     parser.info.app_name = "iGenVar";
-    parser.info.man_page_title = "Short and long Read SV Caller";
-    parser.info.short_description = "Detect junctions in a read alignment file";
+    parser.info.man_page_title = "Short and Long Read SV Caller";
+    parser.info.short_description = "Detect genomic variants in a read alignment file";
     parser.info.version = "0.0.1";
```

Comment:
Should we increase the version at certain steps? Maybe when the modularisation and the big infra stuff is done?

Reply:
Yes, good idea. Let's maybe wait for #81 and #82 until we move to 0.0.2?

Reply:
Jepp and maybe https://github.com/seqan/iGenVar/pull/79 aswell.